README.md

```diff
@@ -9,11 +9,253 @@ GT.M modifications - Sam Habiel.
 # Configuration Instructions
 Import all routines into Cache or GT.M. If you use Cache, make sure that the %ut namespace has been mapped to JDS.
 
-Run the following:
+Run the following to configure and start JDS for the first time, replacing 9080 with the port on which you want JDS to listen:
 
 ```
 D SETUP^VPRJCONFIG
-D GO^VPRJRCL
+D GO^VPRJRCL(9080)
 ```
 
-That's it. JDS is ready to use at this point. You can test ping by hitting `http://<ip address>:9080/ping`.
+This will set up JDS with a single listener on port 9080. From this point forward, you can start JDS with the following:
+
+```
+D START^VPRJ
+```
+
+# Testing JDS
+
+JDS provides a REST endpoint that will confirm that JDS has been successfully configured. In order to access it, launch the following URL:
+
+[http://localhost:9080/ping]
+
+If you have configured JDS to listen on a port other than 9080, modify the above URL (as well as any further example URLs in this documentation) accordingly.
+
+# JDS Documentation
+
+JDS has built-in documentation, accessible at [http://localhost:9080/documentation/index]
+
+This documentation is formatted for [API Blueprint](https://apiblueprint.org), but the API Blueprint renderers I've tried will only render the top-level documentation. I believe there may have been some sort of script or external tooling in the original JDS project for gathering and rendering the documentation into user-friendly HTML. To my knowledge, this has been lost to time.
+
+# GDS Usage Addendum
+
+JDS includes a component called the Generic Data Store, or GDS. This is an extremely powerful JSON database that--unlike the rest of JDS--is usable for storing and searching any JSON data of any format, not limited (as in the other parts of JDS) to a specific JSON schema (such as patient data). As the aforementioned built-in documentation is light in its treatment of GDS, this section will provide a helpful guide for getting started with basic GDS CRUD, housekeeping, and monitoring endpoints.
```

Comment:
I'll have to check a later version of the eHMP code-in-flight to see if there are more updates to that documentation. I remember documenting it, just don't remember where...

Reply:
I just checked. I guess I didn't document this as I thought I did in any release, or maybe my memory is fuzzy since I wrote this over 2 years ago.

Reply:
Would I be correct to presume from this that updates to JDS were (or may have been) done after  the GT.M port?

I've only hacked on the eHMP codebase to a minimal level. I got as far as getting a small portion of the ehmp-ui to come up, but never got the assets to precompile.

Reply:
I pulled the latest eHMP release from OSEHRA's Technical Journal (which is where all the Code In Flight is/was released), I didn't see anything major missing from JDS when I looked. I assume I'm just mis-remembering the documentation I wrote or I wrote it in my dreams one night and it never actually made it to code...